modules/mvt/src/lib/binary-vector-tile/vector-tile-layer.js

```diff
@@ -0,0 +1,79 @@
+// This code is forked from https://github.com/mapbox/vector-tile-js under BSD 3-clause license.
+
+import VectorTileFeature from './vector-tile-feature.js';
+
+export default class VectorTileLayer {
+  constructor(pbf, end) {
+    // Public
+    this.version = 1;
+    this.name = null;
+    this.extent = 4096;
```

Comment:
Is it expected for this to be hardcoded?

Reply:
Again, from the original implementation - I'm not clued up enough to comment